<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAgree</class>
 <widget class="QDialog" name="AddAgree">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Заключить договор</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Заключение договора</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>401</width>
     <height>201</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="opaqueResize">
    <bool>false</bool>
   </property>
   <property name="handleWidth">
    <number>14</number>
   </property>
   <widget class="QSplitter" name="splitter_2">
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Номер договора</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Номер паспорта клиента</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Код филиала</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Вид страхования</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Дата заключения</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Срок заключения (месяцев)</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="opaqueResize">
     <bool>false</bool>
    </property>
    <property name="handleWidth">
     <number>12</number>
    </property>
    <widget class="QLineEdit" name="lineEdit"/>
    <widget class="QLineEdit" name="lineEdit_2"/>
    <widget class="QLineEdit" name="lineEdit_3"/>
    <widget class="QComboBox" name="comboBox">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Машина</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Жизнь</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Жилище</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ответственность</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5"/>
    <widget class="QLineEdit" name="lineEdit_6"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
